packages/engine/src/actions/request.test.js

```
@@ -33,27 +33,31 @@ const mockReqResponses = {
 
 const mockCallRequest = jest.fn();
 const mockCallRequestImp = ({ requestId }) => {
-  return new Promise((resolve, reject) => {
-    if (requestId === 'req_error') {
-      reject(mockReqResponses[requestId]);
-    }
-    resolve(mockReqResponses[requestId]);
-  });
+  if (requestId === 'req_error') throw mockReqResponses['req_error'];
+  return mockReqResponses[requestId];
 };
 
-const pageId = 'one';
 const lowdefy = {
-  callRequest: mockCallRequest,
-  pageId,
+  _internal: {
+    actions: {
+      Request: ({ methods: { request }, params }) => {
+        return request(params);
+      },
+    },
+    blockComponents: {
+      Button: { meta: { category: 'display' } },
+    },
+    callRequest: mockCallRequest,
+  },
 };
 
 const RealDate = Date;
 const mockDate = jest.fn(() => ({ date: 0 }));
 mockDate.now = jest.fn(() => 0);
 
 // Comment out to use console
-console.log = () => {};
-console.error = () => {};
+// console.log = () => {};
+// console.error = () => {};
```

Remove comments to silence errors and logs